### PR TITLE
Release 3.0.0-rc.2

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 3.0.0-rc.2 (2018-11-20)
+
+ - Fixed body cells from overflowing when consumer provides column width. [#13](https://github.com/blackbaud/skyux-grids/pull/13)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: "Fixed (thing) to..."

"Fixed grid cells to properly handle text overflow when column widths are fixed."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+ - Added support for `@skyux/list-builder-common@3.0.0-rc.1`. [#14](https://github.com/blackbaud/skyux-grids/pull/14)
+ - Added sort method to demo. [#18](https://github.com/blackbaud/skyux-grids/pull/18)
```
Comment (Blackbaud-SteveBrush) [inline CHANGELOG.md]: Needs a tad more specificity: "Added column sort function to grid demo."

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Also, the bug fixes should be listed at the bottom of the list.

File: CHANGELOG.md (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: fixed.

**Pull request merged**